.github/workflows/publish.yaml

```diff
@@ -0,0 +1,29 @@
+name: Publish
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+
+jobs:
+  publish-api-doc:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone the module
+        uses: actions/checkout@v2
+
+      - name: Setup Doxygen
+        run: sudo apt install -y doxygen
+
+      - name: Build module API documentation using Doxygen
+        run: |
+            cmake .
+            make module_api
+            sed -e "s%@PROJECT_.\+_DIR@/%%" Doxyfile.API.in > Doxyfile.API
+            doxygen Doxyfile.API
```

Comment:
AFAIR, doxygen emits a couple of warnings. We can add the similar job into `lint.yaml`, which will not publish anything, but will grep warnings and fail if there is one. The idea is borrowed from [here](https://github.com/tarantool/graphql.0/blob/05f39946299cb2f35a97be326b992aace0205eaf/Makefile#L26-L31).

It is just side idea. Feel free to just close this discussion.

Reply:
Okay.